### ISSUE TITLE
Extend MapFragment

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Cemberk/android-maps-extensions#37]
```
This package is missing its own MapFragment class.

I suspect that is because it currently uses the v4 support library. If this 
project switches to the v13 support library, that should enable the project to 
implement its own MapFragment class, extending the one provided in the Google 
library.

```

Original issue reported on code.google.com by `baron.k....@gmail.com` on 21 Nov 2013 at 1:23

**Comment by GoogleCodeExporter** [comment]
```
I have actually added MapFragment very recently.

It is currently on develop branch here: 
http://code.google.com/p/android-maps-extensions/source/detail?r=d3650a09bb567d2
03b49332e0996f1ebbdc52f1c&name=develop and will hit a release 2.0, but you can 
get the latest sources from here or github.
```

Original comment by `maciek.g...@gmail.com` on 22 Nov 2013 at 8:53
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Changing to a more appropriate status. Will set back to Fixed when final 2.0 is 
out.
```

Original comment by `maciek.g...@gmail.com` on 22 Nov 2013 at 9:02
- Changed state: **FixedNotReleased**

**Comment by GoogleCodeExporter** [comment]
Original comment by `maciek.g...@gmail.com` on 27 Dec 2013 at 8:39
- Changed state: **Fixed**
